app/helpers/discovered_hosts_helper.rb

```diff
@@ -65,5 +65,11 @@ def discovery_status_icon(host)
 
     "<span class='glyphicon #{status_glyph}' style='color: #{status_color}'
       title='#{status_message}'/>".html_safe
+
+  end
+
+  def breaking_wrap_wrap(txt, col = 80)
+    txt.gsub(/(.{1,#{col}})( +|$\n?)|(.{1,#{col}})/,
+             "\\1\\3\n")
   end
```

Comment:
Ok wanted to give the facts a bit more length without overflowing, but like you said ill just truncate it.